src/forms/Select.jsx

```diff
@@ -45,7 +42,7 @@ export const SelectInput = (props: Props) => {
 			name={name}
 			error={error}
 			helperText={helperText}
-			{...requiredProps}
+			requiredText={requiredText}
```

Comment:
Looks like you're removing the pattern I was using (maybe due to my misunderstanding of how props work).

What happens if require is `false`? will `requiredText` be set to false? how will it affect the fact that `requiredText` is only [supposed to be a string](https://github.com/meetup/swarm-ui/blob/master/packages/swarm-components/src/Select.jsx#L33)?